lib/curl/curl.nit

```diff
@@ -34,7 +34,7 @@ end
 class Curl
 	super FinalizableOnce
 
-	private var native = new NativeCurl.easy_init
+	var native = new NativeCurl.easy_init is writable
```

Comment:
You should leave these services private. Instead, in `rest.nit` use `intrude import curl` to get access to them. This way the clients don't see internal services that may change at any time!